eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -0,0 +1,5 @@
+<UsageData>
+  <IgnorePatterns>
+    <UsagePattern IdentityGlob="*/*" />
```

Comment:
not **/*?  (recursive dirs)?

Reply:
Based on the docs [here](https://github.com/dotnet/source-build/blob/master/Documentation/planning/arcade-powered-source-build/onboarding/local-onboarding.md#engsourcebuildprebuiltbaselinexml):

> This defines which prebuilt NuGet packages are allowed to be used during the build. Initially, all package IDs and versions are permitted (`*/*`).
>
> The `*/*` glob means "any nupkg id, any version". It will be replaced with more specific rules at a later phase of the implementation plan.

It seems to mean a `name/version` pair rather than a file glob.

Reply:
Yes, this matches the common NuGet identity representation `Microsoft.NETCore.App.Runtime/5.0.0`. (Typical nupkg filenames are ambiguous, using `.` instead of `/`.)